addons/mail/static/src/discuss/core/common/action_panel.xml

```diff
@@ -13,7 +13,7 @@
     <t t-name="mail.ActionPanel.content">
         <div class="o-mail-ActionPanel-header position-sticky top-0 pt-2 pb-1 d-flex flex-column bg-inherit small" t-att-class="{ 'px-1': env.inChatWindow, 'px-2': !env.inChatWindow }">
             <div class="d-flex align-items-baseline gap-1">
-                <button t-if="env.closeActionPanel" class="btn p-1 opacity-25 opacity-100-hover" title="Close panel" t-on-click.stop="env.closeActionPanel">
```

Comment:
IMO we should replace `opacity-25` by `opacity-75` and keep the hover effect. The back button with `opacity-100` looks weird when all the other elements have less opacity.

Reply:
Yeah you're right